profiles/MikhailGulkin.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/95026332/231028777-91783ca3-9253-414d-967c-bd0a459198c7.png",
+  "issueId": 1795,
+  "name": "MikhailGulkin",
+  "username": "Bol4onok"
```

AFAIK, these two should be swapped.
```suggestion
  "name": "Bol4onok",
  "username": "MikhailGulkin"
```